README.md

```diff
@@ -8,11 +8,49 @@ You can also find us at https://meetup.com/utahrust and in the `#utah` channel a
 
 ### Editing the Website
 
+Install Rust
+
+```
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+```
+
+Install Cobalt
+
+```
+cargo install cobalt-bin
+```
+
+Clone the and enter website
+
+```
+git clone git@github.com:utah-rust/utah-rust.github.io.git
+cd ./utah-rust.github.io/
+```
+
+Build the website
+
+```bash
+cobalt build
+```
+
+Preview
+
+```bash
+cobalt serve
+```
+
+Sync the site folder to the webhost
+
+```bash
+ls ./site/
```

Comment:
Does...this actually do any syncing?

Reply:
No, I just put it there as a placeholder and forgot to change it.

Reply:
I'd prefer this either be removed or fixed, then, if it's not going to be helpful. But we're good to merge as soon as this is taken care of!

Reply:
I'll remove this before merge. Thanks!